src/frontend/src/components/Posts/Posts.jsx

```diff
@@ -30,6 +31,7 @@ const REFRESH_INTERVAL = 5 * 60 * 1000; /* refresh data every 5 minutes */
 
 const Posts = () => {
   const classes = useStyles();
+  const [isDesktopView, setDesktopView] = useState(true);
```

Comment:
We had to do something hacky like this for the sticky header views too.  I wonder if we can abstract this out into a hook or context we (re)use at the top level?